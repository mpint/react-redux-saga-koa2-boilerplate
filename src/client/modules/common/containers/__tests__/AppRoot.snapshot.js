import React from 'react';
import renderer from 'react-test-renderer';

import { ConfiguredProvider, wrapWithContext, store, muiTheme, config, router } from '~/lib/test.helpers';

import AppRoot from '../AppRoot';

describe('AppRoot', function() {
  const contextTypes = { muiTheme: React.PropTypes.object, config: React.PropTypes.object, router: React.PropTypes.object };

  it('should render', function() {
    const component = renderer.create(
      wrapWithContext(
        { muiTheme, config, router },
        contextTypes,
        <ConfiguredProvider>
          <AppRoot />
        </ConfiguredProvider>
      )
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
