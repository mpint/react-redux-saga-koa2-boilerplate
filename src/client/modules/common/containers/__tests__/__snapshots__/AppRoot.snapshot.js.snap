exports[`AppRoot should render 1`] = `
<div>
  yoyo
  <div>
     
     
  </div>
  <div
    style={
      Object {
        "bottom": 10,
        "color": "white",
        "fontSize": 10,
        "left": 10,
        "position": "fixed",
        "zIndex": 1500,
      }
    }>
    ver. 0.5.0
     (8658096)
  </div>
</div>
`;
